### DISCUSSION TITLE
自定义顶栏在下滑进入首页后平滑关闭透明填充功能，提升观感。

### DISCUSSION THREAD

**Discussion by MJSummer** [the1812/Bilibili-Evolved#4996] [功能建议 / Ideas]
如题。头图黑色但首页底是白色。进行透明度的顺滑降低能够显著提升顶栏使用体验。
![image](https://github.com/user-attachments/assets/d0d7aa8e-6012-4233-8e1d-49c765ff22ef)
![image](https://github.com/user-attachments/assets/0de0d6a2-b9dc-4d74-9b8a-7ef8136c0664)

**Comment by MJSummer** [comment]
此外视频页顶栏颜色也与主页相割裂。
![image](https://github.com/user-attachments/assets/84d410b3-b148-4c52-9c41-b958fc27ce5d)

**Reply by the1812** [reply]
如果你是指 Logo 的话, 这是因为开了 `使用季节 Logo`, 关掉就行了

**Reply by MJSummer** [reply]
不单指logo。主要还是顶栏的背景色与透明度。如果logo有纯黑白两种就更好了。

**Comment by Geeyun-JY3** [comment]
也不只是首页了，真要做应该是存在顶部横幅的页面。

**Comment by Geeyun-JY3** [comment]
这也算是和首页原版顶栏对齐了。
当然，首页原版顶栏并不会在下滑后平滑关闭透明填充，单纯是下滑一点点就直接换成通用样式的原版顶栏：
![image](https://github.com/user-attachments/assets/a4274eca-c04f-43ee-a66b-b100a5bf2375)
![image](https://github.com/user-attachments/assets/1e7f582e-5a7e-4df5-89fe-685b0f19269b)
